addons/product/models/res_partner.py

```diff
@@ -48,10 +52,14 @@ def _inverse_product_pricelist(self):
                 partner.specific_property_product_pricelist = False if partner.property_product_pricelist.id == default_for_country.id else partner.property_product_pricelist.id
 
     def _commercial_fields(self):
-        return super()._commercial_fields() + ['property_product_pricelist']
+        return [
+            *super()._commercial_fields(),
+            'property_product_pricelist',
+            'specific_property_product_pricelist',
```

Comment:
Isn't `specific_property_product_pricelist` sufficient? It's the only stored one, and writing on `property_product_pricelist` will trigger the write on `specific_property_product_pricelist` through the inverse no? :thinking: